.travis.yml

```diff
@@ -15,7 +16,7 @@ matrix:
       env:
         - COV_CORE_SOURCE=secretsanta
         - COV_CORE_CONFIG=.coveragerc
-    - python: "3.8-dev"
+    - python: "3.8"
       env:
         - COV_CORE_SOURCE=secretsanta
         - COV_CORE_CONFIG=.coveragerc
```

Comment:
Would it be a good idea to run doctest as part of the CI job?

Reply:
yes, we could do that